tgui/packages/tgui/interfaces/GasMixer/types.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file
+ * @copyright 2023
+ * @author Mordent (https://github.com/mordent-goonstation)
+ * @license ISC
+ */
+
+export type GasMixerData = {
+  name: string;
+  mixerid: string;
+  mixer_information: mixerInformation;
+  MAX_PRESSURE: number;
+};
+
+type mixerInformation = {
+  tag: string;
+  timestamp: number;
+  target_pressure: number;
+  pump_status: 'Online' | 'Offline';
+
+  in1: airInfo;
+  in2: airInfo;
+
+  i1trans: number; // Input 1 ratio
+  i2trans: number; // Input 2 ratio
```

Comment:
Why not give a better name if a comment is needed to help understand?

Reply:
Name came from mixer.dm which at first I thought I couldn't touch until I realized it's used exclusively by the gas mixer